web/pages/requests.tsx

```diff
@@ -112,6 +127,47 @@ export const getServerSideProps = async (
   const currentPage = parseInt(page as string, 10) || 1;
   const pageSize = parseInt(page_size as string, 10) || 25;
 
+  const { data: orgLayout, error: organizationLayoutError } =
+    await supabaseServer
+      .from("organization_layout")
+      .select("*")
+      .eq("organization_id", orgId)
+      .eq("type", "requests")
+      .single();
+
+  if (!orgLayout || organizationLayoutError) {
+    return {
+      props: {
+        user: user,
+        currentPage,
+        pageSize,
+        sort: {
+          sortKey: sortKey ? (sortKey as string) : null,
+          sortDirection: sortDirection
+            ? (sortDirection as SortDirection)
+            : null,
+          isCustomProperty: isCustomProperty === "true",
+        },
+        initialRequestId: requestId ? (requestId as string) : null,
+        currentFilter: null,
+        orgLayout: orgLayout ?? null,
+      },
+    };
+  }
+
+  const filterId = context.query.filter as string;
+
+  const filters: OrganizationFilter[] =
+    orgLayout.filters as OrganizationFilter[];
+  const layout: OrganizationLayout = {
+    id: orgLayout.id,
+    type: orgLayout.type,
```

Comment:
nit: in the future, we should name these something other tha. just `type`. `orgLayoutType` or `orgLayoutPage` would be a little better. no need to change for this PR